### ISSUE TITLE
Feature request: add vapoursynth support

### ISSUE THREAD

**Issue by s416058** [gnome-mpv/gnome-mpv#379]
It would be very nice if you could use vapoursynth in gnome mpv. I thought that I could compile the program myself with this option. Unfortunately, I can't do it.

**Comment by gnome-mpv** [comment]
If you compiled mpv with vapoursynth support, it should work fine in gnome-mpv.

**Comment by s416058** [comment]
I tried but it didn't work. I added --enable-vapoursynth flag to the building script, a deb package for mpv and then installed a development package for vapoursynth.
After compiling I installed all built packages for mpv (mpv libmpv1 libmpv-dev) and for mpv vapoursynth it works fine, but doesn't for gnome-mpv.
I don't know how to get an similar output to mpv by gnome-mpv to figure out if vapoursynth is detected by it.

**Comment by gnome-mpv** [comment]
How are you setting the vapoursynth filter in gnome-mpv?

Putting `vf=vapoursynth=/path/to/script.py` in the config file and loading it in the preferences dialog seems to work fine for me.

**Comment by s416058** [comment]
Ok, now I see where the problem is. I used some configuration from the Internet for vapoursynth in mpv, which has a path was set by default:
`vapoursynth=~~/motioninterpolation.vpy:4:4`
Having access to logs I noticed that gnome-mpv (libmpv?) is not able to decode this path.
`[vapoursynth] [Errno 2] No such file or directory: 'motioninterpolation.vpy'`
After changing to a more understandable path for the program:
`~/.config/mpv/motioninterpolation.vpy `
Everything works as intended.

Thank for yout help!
